minitests/roi_harness/basys3.sh

```diff
@@ -1,6 +1,8 @@
 # XC7A35T-1CPG236C
 export XRAY_PART=xc7a35tcpg236-1
 export XRAY_PINCFG=BASYS3-SWBUT
+export XRAY_DIN_N_LARGE=17
+export XRAY_DOUT_N_LARGE=17
```

Comment:
why 16? Originally the large target was using 8 ins and 8 outs

Reply:
basys harness uses 16 LED, 16 switches, and a UART RX/TX (hence 17)

Reply:
ok, this seems to be correct. We have 16 LEDs/Swiches + UART